### DISCUSSION TITLE
Oled displays lifetime

### DISCUSSION THREAD

**Discussion by pike2k** [olikraus/u8g2#1657] [General]
Hi. I've picked up somewhere that the lifetime of these small OLED displays are (quite?) limited.
But I have not seen any numbers. So what happens with them over time?
And does it happen only when in use, or also when not in use?

**Comment by olikraus** [comment]
Lifetime of operation (OLED is powdered and shows some content) is the time in hours until brightness has reached 50% of the initial value. For a good industrial yellow OLED, this can be up to 50000 hours. White OLED will only have 20000 hours. Low cost OLED from diy market places might have much lesser lifetime.

**Comment by swarm38** [comment]
> Hi. I've picked up somewhere that the lifetime of these small OLED displays are (quite?) limited. But I have not seen any numbers. So what happens with them over time? And does it happen only when in use, or also when not in use?

Hi.
I use the OLED display in my project for a car. Display SH1106, bought on aliexpress for 5 bucks, worked 3 years in the car and about 50,000 kilometers, at maximum brightness. The test by filling the working area showed that fixed pixels degraded most of all, losing brightness, but this is completely invisible during operation. I hope my information will be useful.
via GoogleTranslate
![imgonline-com-ua-Resize-McPnlfpvJsXPI5i](https://user-images.githubusercontent.com/72743272/142726483-d2039de4-53fd-4baa-bb32-726fd82e25dd.png)
.
